.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,221 @@
+        3. TI(s) drafts that includes:
+          - Problem Statement in the form below
+          - Proposed work and other work considered
+          - Benefit to OSSF and ecosystem
+          - Approximate cost
+        4. TI submits this issue to the TAC. This will be treated like a CFP. Respondents will be notified in writing, and approved proposals will be published. For the first year, grants will be reviewed as they come in and will be reviewed until funding runs out for all qualified grant applications. 
+        5. TAC evaluates the technical merit of the proposal and checks that its goals within OSSF and ecosystem seem reasonable and achievable. (See TAC evaluation below)
+        6. TI(s) iterate on the proposal with the TAC until approved or it’s clear that the proposal is unreasonable.
+        7. TAC brings approved proposals to the funding workflow (see below).
+        8. Steps 6-7 are repeated until approved or seen as unreasonable.
+        9. The OpenSSF Staff will distribute funds. 
+        10. TAC will review funding initiative status on milestones. If a funding initiative has more than one milestone, funds may be dispersed after each milestone is achieved (funding initiatives asking for more funds should most likely have more milestones for fund releasing). If the milestone is reached, the TAC/Staff agree that Staff should disperse funds for additional milestones(s).
+  - type: markdown
+    attributes:
+      value: |
+        **TAC review:**
+        - Is this request in line with the [TAC Technical Vision](https://github.com/ossf/tac/blob/main/technical-vision.md)?
+        - Is this request in line with the OpenSSF [MVVS](https://openssf.org/about/), and soon to be [finalized R](https://docs.google.com/document/d/1UoQudHQuaXNzakhOYbAS3IceI9TkSSR6N0bgm1fTTK0/edit#heading=h.493lq0mo4y4f)? (this is how GB/GC communicate “top-down” initiatives, like the SOSS Task Force).
+        - Does the requester(s) have an accepted [TI lifecycle phase](https://github.com/ossf/tac/tree/main/process) in good standing?
+          - Does the request meet the “get” of the associated TI lifecycle phase?
+          - Within general funding limits for the TIs at the respective lifecycle (e.g. Sandbox should generally ask for less than Incubating, which should ask for less than Graduated)
+          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
+        - What will be the milestone review requested and when?
+        - Final decision is open discussion and a majority consensus vote. (Alternatively, could have a private form vote)
+  - type: markdown
+    attributes:
+      value: |
+        **Finance review:**
+        - Who will approve this request?
+          - GM up to $100k, B&F committee up to $250k, GB above $250k
```

Comment:
How does this work with the milestone structure - if a project has 3 milestones of $100k each, does that mean the  total amount goes to the GB for approval first?

Reply:
Yes. Since the funding request would be for a total of $300k, that amount is higher than the delegated authority.